cpp/oneapi/dal/test/engine/common_dpc.cpp

```diff
@@ -37,22 +37,26 @@ static bool check_if_env_knob_is_enabled(const char* env_var) {
     }
 }
 
-static bool check_if_env_overrides_fp64_settings() {
+[[maybe_unused]] static bool check_if_env_overrides_fp64_settings() {
     return check_if_env_knob_is_enabled("OverrideDefaultFP64Settings");
 }
 
-static bool check_if_env_forces_dp_emulation() {
+[[maybe_unused]] static bool check_if_env_forces_dp_emulation() {
     return check_if_env_knob_is_enabled("IGC_EnableDPEmulation") ||
            check_if_env_knob_is_enabled("IGC_ForceDPEmulation");
 }
 
 bool device_test_policy::has_native_float64() const {
+#ifdef ONEDAL_DISABLE_FP64_TESTS
+    return false;
+#else
```

Comment:
Why this is embedded to `has_native_float64()` function instead of creating another one like `is_float64_enabled()`?

Reply:
Because this function is already used in almost every test.

Reply:
Yes, but we can easily replace `SKIP_IF(this->not_float64_friendly());` by `SKIP_IF(this->not_float64_friendly() || !this->is_float64_enabled());`. I think that having a separate function in this case is more accurate, because float64 native support and test configs are different things. But this is a minor comment.